configure.ac

```diff
@@ -2519,7 +2520,6 @@ AC_MSG_RESULT([$enable_slower_safety])
 
 if test "$enable_slower_safety" = "yes"
 then
-  AX_CHECK_COMPILE_FLAG([-D_FORTIFY_SOURCE=3], [BASECFLAGS="$BASECFLAGS -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3"], [AC_MSG_WARN([-D_FORTIFY_SOURCE=3 not supported])])
```

Comment:
@nohlson you can remove whole code block for testing purpose.

Reply:
Removed. Ready to test with buildbots